index.html

```diff
@@ -22,31 +22,34 @@
   <body>
     <div class="page-container">
       <div class="header1">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
-      </header>
-    </div>
-<div class="header2">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><ul class="navlist">
-          <li> <a href="#">Portfolio</a></li>
-          <li> <a href="#">About</a></li>
-          <li> <a href="#">Contact</a></li>
-        </ul></nav>
-      </header>
-    </div>
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav><i class="fa-solid fa-bars hambrgricon" id="hmbg"></i></nav>
+        </header>
+      </div>
+      <div class="header2">
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav>
+            <ul class="navlist">
+              <li class="linkitems"><a href="#portfolio">Portfolio</a></li>
+              <li class="linkitems"><a href="#about">About</a></li>
+              <li class="linkitems"><a href="#contact">Contact</a></li>
+            </ul>
+            <i class="fa-solid fa-times closeicon" id="closebtn"></i>
+          </nav>
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** As this comment was not mentioned by the previous reviewer, kindly make sure your mobile menu follows the styling like on the Figma design(e.g.: adding the blur effect on the background, closeMenu button color, ... ).

| Figma Design | Your Design |
| ------------------- | -------------------- |
| ![mmf1](https://user-images.githubusercontent.com/83042742/181495063-2f4cae1a-fa8b-4ddb-8849-efae7859a5ca.png) | ![Screenshot from 2022-07-28 13-28-58](https://user-images.githubusercontent.com/83042742/181495116-ad09e232-a2ba-4028-9426-b95e749b7e07.png) |
 

- [ **OPTIONAL** ] **RECOMENDED:**  What do you think if you also add a styling of the mobile menu on screen width less than `375px` as any styling is provided to make your webpage looks more professional. Below is how your mobile menu looks like:
![Screenshot from 2022-07-28 13-37-10](https://user-images.githubusercontent.com/83042742/181496193-694b35ed-7d83-4dd8-8b8f-4c06f5bd687d.png)